admin_guide/scheduling/node_affinity.adoc

```diff
@@ -18,7 +18,7 @@ For example, you could configure a pod to only run on a node with a specific CPU
 
 There are two types of node affinity rules: _required_ and _preferred_.
 
-Required rules *must* be met before a pod can be scheduled on a node. Preferred rules specify that, if the rule is met, the scheduler will try to enforce the rules, but does not guarantee enforcement.
+Required rules *must* be met before a pod can be scheduled on a node. Preferred rules specify that, if the rule is met, the scheduler will try to enforce the rules, but does not guarantee enforcement. If a pod has an affinity rule for which there is no node with a matching label, that pod can be scheduled on a node that does not have a label 
```

Comment:
@mburke5678  
when no node meet the node affinity rule,
if it is required, pod is not scheduled, in pending, 
if it is preferred, all schedulable nodes are treated equally from node affinity point.